src/FakeItEasy/Creation/IFakeAndDummyManager.cs

```diff
@@ -13,7 +13,7 @@ internal interface IFakeAndDummyManager
         /// <param name="typeOfDummy">The type of dummy to create.</param>
         /// <returns>The created dummy.</returns>
         /// <exception cref="FakeItEasy.Core.FakeCreationException">The current IProxyGenerator is not able to generate a fake of the specified type and
-        /// the current IFakeObjectContainer does not contain the specified type.</exception>
+        /// the current IDummyValueCreationSession does not contain the specified type.</exception>

```

Comment:
Close, and less crucial because it's internal, but I think I'd still adjust this comment. This is a little more difficult becuase the comment had more problems than just referring to `IFakeObjectContainer`.

You can see that [CreateDummy](https://github.com/FakeItEasy/FakeItEasy/blob/56b4689254c3fe32a991a84278ba2de3b56e8bc0/src/FakeItEasy/Creation/DefaultFakeAndDummyManager.cs#L28) calls the session's [TryResolveDummyValue](https://github.com/FakeItEasy/FakeItEasy/blob/fdeb2b1f8a9e0dd363233befa97ff5fa9985722f/src/FakeItEasy/Creation/DummyValueCreationSession.cs#L38) method, which makes sure that the type isn't recursive and then defers to [TryResolveDummyValueWithAllAvailableStrategies](https://github.com/FakeItEasy/FakeItEasy/blob/fdeb2b1f8a9e0dd363233befa97ff5fa9985722f/src/FakeItEasy/Creation/DummyValueCreationSession.cs#L71), which looks for a cached Dummy resolution strategy, and if it doesn't find one, looks through [all its available strategies](https://github.com/FakeItEasy/FakeItEasy/blob/fdeb2b1f8a9e0dd363233befa97ff5fa9985722f/src/FakeItEasy/Creation/DummyValueCreationSession.cs#L29-L34) in turn, using the first one that works. This brings us to the list of strategies in [How are the Dummies made?](http://fakeiteasy.readthedocs.io/en/stable/dummies/#how-are-the-dummies-made).

But the comment makes it seem like we're all about Fake creation, and failing that doing something with some registered types in the (was `IFakeObjectContainer`, now `IDummyValueCreationSession`). And the session doesn't really contain types as such. Probably the container didn't either.

To make matters worse, this comment is on an interface, and really, how can it guarantee what implementers are going to do in order to create Dummy?
For my money, I think I'd just indicate that we throw the exception if the we're unable to create a Dummy of the requested type. It's simpler and less likely to become false in the future…

Reasonable?